Core/fft_mt_r2iq.cpp

```diff
@@ -200,42 +186,44 @@ void fft_mt_r2iq::Init(float gain, int16_t **buffers, float** obuffers)
 		}
 
 		filterplan_t2f_c2c = fftwf_plan_dft_1d(halfFft, pfilterht, filterHw[0], FFTW_FORWARD, FFTW_MEASURE);
+		float *pht = new float[halfFft / 2 + 1];
 		for (int d = 0; d < NDECIDX; d++)
 		{
-			float* pht;
+			const float stopdb = 120.0f;
 			switch (d)
 			{
 			case 5:
-				pht = FIR5;
+				KaiserWindow(halfFft / 2 + 1, stopdb, 0.8f/32.0f, 1.0f/32.0f, pht);
```

Comment:
KaiserWindow(**halfFft / 4 + 1**, stopdb, 0.8f/**64.0f**, 1.0f/**64.0f**, pht);
halfFft / 4 + 1 as ht must be equal to 1/4 of the FFT used (4096) to simplify convolution code.
I use 64.0 as sampling... Otherwise you can use  0.4f/32.0f   0.5f/32.0F the filter is used referred to LO +0.4, LO -0.4 = 0.8 span..
Please made these correction for all the cases of the switch.